src/v/raft/tests/append_entries_test.cc

```diff
@@ -761,3 +763,38 @@ FIXTURE_TEST(test_linarizable_barrier_single_node, raft_test_fixture) {
         BOOST_REQUIRE(are_logs_the_same_length(logs));
     }
 };
+
+FIXTURE_TEST(test_big_batches_replication, raft_test_fixture) {
+    raft_group gr = raft_group(raft::group_id(0), 1);
+    gr.enable_all();
+    auto leader_id = wait_for_group_leader(gr);
+    auto leader_raft = gr.get_member(leader_id).consensus;
+
+    bool success
```

Comment:
I think a better test is to move this to ducktape and have 

1. a constant 1KB workload 
2. intermingle a 15MB payload like 5 times
3. once the big payload finishes, you stop the constant 1KB workload

Reply:
i should say in addition-to this test.

Reply:
Unfortunately Noah's PR with verifiable consumer hasn't yet landed.